rundeckapp/grails-app/controllers/rundeck/controllers/EditOptsController.groovy

```diff
@@ -35,6 +37,7 @@ import java.util.regex.PatternSyntaxException
 class EditOptsController extends ControllerBase{
     static Logger logger = LoggerFactory.getLogger(EditOptsController)
     def FrameworkService frameworkService
+    DataManager rundeckDataManager
```

Comment:
Rather than using the DataManager let use the UserDataProvider directly.

Reply:
Done